src/bundle/Resources/public/js/CKEditor/custom-tags/ui/custom-tag-form-view.js

```diff
@@ -0,0 +1,208 @@
+import View from '@ckeditor/ckeditor5-ui/src/view';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import LabeledFieldView from '@ckeditor/ckeditor5-ui/src/labeledfield/labeledfieldview';
+
+import Model from '@ckeditor/ckeditor5-ui/src/model';
+import Collection from '@ckeditor/ckeditor5-utils/src/collection';
+import { createLabeledInputText, createLabeledDropdown } from '@ckeditor/ckeditor5-ui/src/labeledfield/utils';
+import { addListToDropdown } from '@ckeditor/ckeditor5-ui/src/dropdown/utils';
+
+import { createLabeledInputNumber } from '../../common/input-number/utils';
+import { createLabeledSwitchButton } from '../../common/switch-button/utils';
+
+class CustomTagFormView extends View {
+    constructor(props) {
+        super(props);
+
+        this.locale = props.locale;
+
+        this.saveButtonView = this.createButton('Save', null, 'ck-button-save', 'save-custom-tag');
```

Comment:
translations?

Reply:
I want to add translations in the design phase, I'm not sure if this will be a word or an icon.